templates/base.html

```diff
@@ -1,5 +1,6 @@
 <!doctype html>
 {% load static bootstrap4 %}
+{% bootstrap_javascript jquery='full' %}
```

Comment:
so this definitely works just having it in the base? I thought we had tried that and needed it in the child pages or the javascript controls didn't work...

Reply:
Earlier it was working on some pages, for some sections, but not all the time. There was actually some javascript that was being loaded in by a vue entrypoint in one of the sub templates that was messing up the javascript that had been loaded like this... another change that I made in this PR was moving some of the extra_javascript stuff out of the block content sections in various templates. After sorting through that stuff, things seemed to be working, but bootstrap and the extra javascript libraries were being loaded multiple times, once each for each time this bootstrap_javascript tag was included. It does look to work like this.